packages/core/src/interfaces/charts.ts

```diff
@@ -104,6 +104,20 @@ export interface BaseChartOptions {
 		 */
 		scale?: object;
 		/**
+		 * use a carbon dataviz preset color palette
+		 * put the index (selection of which variant)
+		 */
+		pairing?: {
+			/**
+			 * the number of data groups/categories, only if differ from given data
+			 */
+			numberOfGroups?: number;
```

Comment:
I'd rather this was called colorGroups, colorVariants or something that makes it more intuitive that its the number of **colors** rather than groups that you are overriding in the chart. Since i think you could just **choose** to use a bigger colorset for visual parity with the rest of the dashboard/page,  it makes this config seem a bit misleading to me.

Reply:
I know it's not a good name for this config, I'd like to see if anyone comes up with a better idea for this. I thought about `colorGroups`, it seems more like the input is an array of string, not an integer. It might be a bit confused about the type.

Reply:
variants? numberOfVariants? 🤔

Reply:
I'll take `numberOfVariants`.